llvm/test/Analysis/CtxProfAnalysis/inline.ll

```diff
@@ -0,0 +1,101 @@
+; RUN: rm -rf %t
+; RUN: split-file %s %t
+; RUN: llvm-ctxprof-util fromJSON --input=%t/profile.json --output=%t/profile.ctxprofdata
+
+; RUN: opt -passes='module-inline,print<ctx-prof-analysis>' %t/module.ll -S \
+; RUN:   -use-ctx-profile=%t/profile.ctxprofdata -ctx-profile-printer-level=json \
+; RUN:   -o - 2> %t/profile-final.txt | FileCheck %s
+; RUN: %python %S/json_equals.py %t/profile-final.txt %t/expected.json
+
+; CHECK-LABEL: @entrypoint
+; CHECK-LABEL: yes:
+; CHECK:         call void @llvm.instrprof.increment(ptr @entrypoint, i64 0, i32 3, i32 1)
```

Comment:
Naive question, is it intended that `llvm.instrprof.increment` [intrinsic](https://llvm.org/docs/LangRef.html#llvm-instrprof-increment-intrinsic) is present in a ctx profile use IR?

Reply:
yes. we keep the instrumentation until we flatten the profile and then lower the flattened profile to `MD_prof`. The instrumentation will serve as a means to associate counter values in the contextual profile to BBs.